.infra/application.properties

```diff
@@ -9,7 +9,7 @@ debezium.source.database.password=%database_pass%
 debezium.source.database.dbname=%database_dbname%
 debezium.source.database.server.name=api
 debezium.source.table.include.list=public.comment,public.user_comment,public.comment_mention,public.source_request,public.post,public.user,public.post_report,public.source_feed,public.settings,public.reputation_event,public.submission,public.user_state,public.notification_v2,public.source_member,public.feature,public.source,public.post_mention,public.content_image,public.comment_report,public.user_post,public.banner,public.post_relation,public.marketing_cta,public.squad_public_request,public.user_streak,public.bookmark,public.user_company,public.source_report,public.user_top_reader,public.source_post_moderation
-debezium.source.column.exclude.list=public.post.tsv,public.post.placeholder,public.source.flags,public.user_top_reader.image
+debezium.source.column.exclude.list=public.post.tsv,public.post.placeholder,public.source.flags,public.user_top_reader.image,public.user.cioRegistered,public.user.acceptedMarketing,public.user.followingEmail,public.user.notificationEmail
```

Comment:
We don't need to track changes from any of these columns.

Reply:
Are you sure though?
For example acceptedMarketing should trigger userUpdated no? to sync them to CIO (when they not in your flow)

Reply:
Ah, right. We have these props in CIO, they should definitely be synced. Let me check the worker.

Reply:
Maybe I should not add these columns, but only when running the initial rollout.

Reply:
I think it does, didn't we fix that issue a while back by finally updating debezium?

Reply:
It's not an issue just misunderstanding of what this config does from our side. But it's better to check with adhoc infra any change in debezium props